Product/Production/Services/AuditRepositoryCore/src/main/java/gov/hhs/fha/nhinc/auditrepository/nhinc/AuditRepositoryOrchImpl.java

```diff
@@ -116,7 +116,9 @@ public AcknowledgementType logAudit(LogEventSecureRequestType mess, AssertionTyp
             if (activeParticipant != null) {
                 eventUserId = activeParticipant.getUserID();
                 if (eventUserId != null && !eventUserId.equals("")) {
-                    auditRec.setUserId(eventUserId);
+                    /* this value is temporary set to empty strings and due to length of characters are exceeding
+                     more than 100*/
+                    auditRec.setUserId("");
```

Comment:
We have to re-design audit database and the userName is defined to 100 characters which is enough if we populate the HumanRequestor userId. Currently it always populates the first ActivaParticipant userId for both initiating and Responding Gateway and in this case the service endpoint is the source ACtivePartcicipant UserId. This will be addressed as part of FHAC-511 which is database design for audit db.